critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,25 @@
+from hashlib import md5
+from flask import Blueprint, render_template
+from flask import current_app
+from critiquebrainz.db import users as db_users
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/')
+def moderators():
+    admins = current_app.config['ADMINS']
+    moderators = db_users.get_many_by_mb_username(admins)
+    config_list = [moderator['display_name'] for moderator in moderators]
+    # list of MB users without CritiqueBrainz account
+    missing_users = list(set(admins) - set(config_list))
```

Comment:
I'm talking about naming of the variables.

Here's how I would write that view:
```python
@moderators_bp.route('/')
def moderators():
    mb_usernames = set(current_app.config['ADMINS'])
    mods_data = []
    mb_usernames_found = set()
    for mod in db_users.get_many_by_mb_username(mb_usernames):
        mods_data.append({
            'musicbrainz_id': mod['musicbrainz_id'],
            'avatar_url': get_gravatar_url(
                use_email=mod['show_gravatar'],
                email=mod['email'],
                alt=mod['id'],
            ),
        })
        mb_usernames_found.add(mod['musicbrainz_id'])
    for username in mb_usernames - mb_usernames_found:  # Moderators without a CB account
        mods_data.append({
            'musicbrainz_id': username,
            'avatar_url': get_gravatar_url(alt=username),
        })
    return render_template('moderators/moderators.html', moderators=mods_data)
```